config/routes.rb

```diff
@@ -101,6 +100,8 @@
     end
   end
 
-  #Rescue not found routes
-  get '*not_found', to: 'application#not_found'
+  # Rescue not found routes
+  get '*not_found', to: 'application#not_found', constraints: -> (req) do
+    !(req.fullpath =~ /^\/rails\/active_storage\/.*/)
```

Comment:
Why is this necessary?

Reply:
That's the route where the uploads can be accessed, it's a wrapper that uses disk storage on development and the configured provider on production (an S3 bucket for example)

This "not found" route was preventing the ActiveStorage controller routes to work, since the engine adds them at the very end of the list.